nanshe_workflow/par.py

```diff
@@ -123,6 +129,71 @@ def get_client(profile):
     return(client)
 
 
+@contextmanager
+def get_executor(client):
+    executor = client.become_dask()
+    try:
+        yield executor
+    finally:
+        executor.shutdown()
```

Comment:
Apparently we need both to avoid warnings from `distributed`. This is addressed in the follow-up PR ( https://github.com/nanshe-org/nanshe_workflow/pull/74 ).